apps/statushq/lib/statushq_web/controllers/admin/status_page_controller.ex

```diff
@@ -18,7 +18,7 @@ defmodule StatushqWeb.Admin.StatusPageController do
   plug :load_active_incidents when action not in [:index, :new, :create]
 
   def index(conn, _params) do
-    status_page = SPM.get_page(conn.assigns.current_user)
```

Comment:
This should match with https://github.com/statuspal/statuspal/blob/master/apps/statushq/lib/statushq/spm.ex#L49, using the user ID looks like it would actually break things